man/dockerd.8.md

```diff
@@ -441,7 +441,7 @@ device. And devices automatically go away when last user of the device
 exits.
 
 For example, when a container exits, its associated thin device is removed. If
-that device has leaked into some other mount namespace and can't be removed,
+that device has leaked into some other mount namespaces and can't be removed,
```

Comment:
But "some" does not have to be plural; "some other person", "some other house"? The mount only has to leak into a _single_ other namespace for this problem to occur.